tests/tailwind4.test.js

```diff
@@ -190,32 +190,169 @@ describe("Tailwind 4", function () {
     });
 
     describe("@source", () => {
-        it("should parse @source with a valid URL", () => {
-            const tree = toPlainObject(parse("@source url('https://example.com/styles.css');"));
```

Comment:
I don't think `url()` is valid here, it isn't mentioned in [the docs](https://tailwindcss.com/docs/detecting-classes-in-source-files), also I can't imagine what the semantics would be for that.